### ISSUE TITLE
CopyProtectionBreak freezes Outlook.com

### ISSUE THREAD

**Issue by EstherMoellman** [Dook1/AutocopySelection2keyboard#7]
Hi,

First, thank you for your great add-on, extremely useful to me.

I would like to report an issue with https://outlook.live.com
When "CopyProtectionBreak" is selected at options, is impossible to replay emails at outlook.com, the message box freezes.
This issue never existed before, I only can say that if I uncheck "CopyProtectionBreak", the message box unfreezes and works fine at outlook.com.
The problem is that I need to have "CopyProtectionBreak" selected, because I use this option for lot of other websites. And it is very uncomfortable to uncheck it every time I write an email at outlook.com.

I am on Firefox Nightly, latest version 20181205102000 / Windows 10 latest version.

I hope you can solve the issue... thank you in advance!

**Comment by Dook1** [comment]
I checked. Everything is working fine. "Replay" is working.

**Comment by EstherMoellman** [comment]
Hi @Dook1 , thank you for your replay.

I tested again, now on FF Nightly clean install, zero add-ons, zero-configurations, everything at FF' default settings, just autocopyselection2 installed, default options + "CopyProtectionBreak" selected... and the issue happens.

Here is the detailed way to reproduce the issue (FF Nightly, Windows 10):
1) Login outlook.com
2) Click "replay" at any message
3) Don't write anything, just try to mouse-click inside the message. In other words, by mouse clicking try to put the cursor inside the replayed message and then try to write

... if you do that, you will see the issue: The message box freezes! It is impossible to write inside the replayed box message.
But if you uncheck "CopyProtectionBreak", everything works fine.

Another way to reproduce the issue, is doing the same, but after clicking "replay", start writing something at the beginning of the box message, it will work... but if you stop writing, and you mouse-click inside the received message, the message box will freeze.

Is the mouse clicking inside the message, what triggers the issue. Every time you mouse click inside the original replied message, or every time you put the cursor inside the original message, the box message will freeze. 
And again, if you uncheck "CopyProtectionBreak", everything works fine.

**Comment by Dook1** [comment]
Tested. Everything works. FF65.0b1

**Comment by EstherMoellman** [comment]
Well, I confirm you, is not working. If you follow the steps I explained, you will see the issue. I can make a video for you if you want.

**Comment by Dook1** [comment]
I followed your steps. 
Make a video.

**Comment by EstherMoellman** [comment]
@Dook1 ,

Here is the video I did on FF Nightly, clean install (FF' default settings), just your add-on installed + "CopyProtectionBreak" selected.

At the first part of the video I opened your email, clicked "replay", and clicked inside your message... you will see at the video that the box message is frozen, I only was able to write where I clicked first, I couldn't move from there, everything was frozen.

Then, I went to your add-on, unchecked "CopyProtectionBreak", went back to outlook.com, I did a page refresh, again opened your email, clicked "replay", and everything worked perfect, I was able to click and write everywhere.

Attached you will see the video: https://streamable.com/qrfaf

**Comment by Dook1** [comment]
I do not see any freezes on your video. In any case, the problem is in your system. I checked again, i followed your video - everything works fine for me.

**Comment by EstherMoellman** [comment]
1) My system has no problems at all.
2) I tested in other computer before posting the issue. And the issue always is there.
3) At the video, you perfectly can see the box message freezing, because I mouse-clicked many times (mouse click appears with a yellow shadow) and the mouse cursor appears only the first time. After that, mouse-click disappears, is frozen.
4) By "frozen" I mean I can write, but I can't move from where mouse was clicked, from where cursor was located. Again, is the mouse-clicking that triggers the issue, allowing to write, but not allowing to click or to put the cursor in other part of the message.
5) When "CopyProtectionBreak" is unchecked, is perfectly visible at the video that all mouse clicks are working fine, and I can write wherever I want.

I reported you the issue.
I explained you step by step how to reproduce it.
I posted a video.
It seems that you chose to blame my system for the issue, or to not believe me about the issue. OK! This is your right.
I am done, I finish here my participation.
